src/index.ts

```diff
@@ -26,12 +27,37 @@ const main = async () => {
 		client.logger.info('Logging in');
 		await client.login();
 		client.logger.info('logged in');
-		console.log(container.metaData);
-	} catch (error) {
+    sendReminder(16, 0, 0)
+  } catch (error) {
 		client.logger.fatal(error);
 		client.destroy();
 		process.exit(1);
 	}
 };
 
 main();
+
+/**
+ * sends reminders at a certain time everyday in a specific channel
+ * @param hour 
+ * @param minute 
+ * @param second 
+ */
+async function sendReminder(hour:number, minute: number, second: number) {
+  while (true) {
+    let now:any = new Date();
+    let then:any = new Date(now.getFullYear(), now.getMonth(), now.getDate(), hour, minute, second, 0)
+    let millisTill:number = then - now;
```

Comment:
inferenced type

Reply:
nope it returns a number (ms between the two times)